t/apicast-policy-buffering.t

```diff
@@ -0,0 +1,164 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+require("http_proxy.pl");
+
+sub large_body {
+  my $res = "";
+  for (my $i=0; $i <= 1024; $i++) {
+    $res = $res . "1111111 1111111 1111111 1111111\n";
+  }
+  return $res;
+}
+
+$ENV{'LARGE_BODY'} = large_body();
+
+require("policies.pl");
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: default configuration (buffered)
+--- configuration
+{
+  "services": [
+    {
+      "backend_version":  1,
+      "proxy": {
+        "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/",
+        "proxy_rules": [
+          { "pattern": "/", "http_method": "GET", "metric_system_name": "hits", "delta": 2 }
+        ],
+        "policy_chain": [
+          {
+            "name": "buffering",
+            "version": "builtin",
+            "configuration": {}
+          },
+          {
+            "name": "apicast",
+            "version": "builtin",
+            "configuration": {}
+          }
+        ]
+      }
+    }
+  ]
+}
+--- backend
+location /transactions/authrep.xml {
+  content_by_lua_block {
+    ngx.exit(200)
+  }
+}
+--- upstream
+location / {
+  proxy_pass http://127.0.0.1:19113;
```

Comment:
This is smart. My concern is about conflicts with the port number. Can we get this to be random?